src/Contract.php

```diff
@@ -68,8 +78,7 @@ public function sign(DocumentSignInterface $documentSign)
 
         $document = $documentSign->create(storage_path($this->file), Str::replaceFirst('contracts/', '', $this->file));
         $this->external_id = $document['id'];
-
-        $documentSign->signer('test@live.com')->sign($document['id'], 'Rentbay', 'noreply@rentbay.nl');
+        $documentSign->signer($this->user->email)->sign($document['id'], 'Rentbay', 'noreply@rentbay.nl');
```

Comment:
documentSign->create might fail. This code should have a try catch, and log exceptions